src/components/users/users.ts

```diff
@@ -163,31 +204,21 @@ export async function listUsers(ctx: IContext, params: IParameters): Promise<IRe
     CLOUD_CONTROLLER_GLOBAL_AUDITOR,
   );
 
-  const [isManager, isBillingManager, organization, users] = await Promise.all([
+  const [isManager, isBillingManager, organization, userOrgRoles] = await Promise.all([
     cf.hasOrganizationRole(params.organizationGUID, ctx.token.userID, 'org_manager'),
     cf.hasOrganizationRole(params.organizationGUID, ctx.token.userID, 'billing_manager'),
     cf.organization(params.organizationGUID),
     cf.usersForOrganization(params.organizationGUID),
   ]);
 
-  const usersWithSpaces = await Promise.all(users.map(async (user: IOrganizationUserRoles) => {
-    const userWithSpaces = {
-      ...user,
-      spaces: new Array(),
-    };
+  const spacesVisibleToUser = await cf.spaces(params.organizationGUID);
```

Comment:
Since this doesn't rely on anything returned in the promises in L207, we could group L214 in with it. As it is, we're performing an extra await for no extra value. Unless there's a reason this has to be separated?

Reply:
Yep, you're absolutely right - this is just sequential for no reason. I'll update it.